maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -3370,16 +3362,15 @@ public void setSystemPropertyVariables( Map<String, String> systemPropertyVariab
         this.systemPropertyVariables = systemPropertyVariables;
     }
 
-    public File getSystemPropertiesFile()
-    {
-        return systemPropertiesFile;
-    }
+    /**
+     * List of System properties, loaded from a file, to pass to the JUnit tests.
+     *
+     * @since 2.8.2
+     */
+    public abstract File getSystemPropertiesFile();
 
     @SuppressWarnings( "UnusedDeclaration" )
-    public void setSystemPropertiesFile( File systemPropertiesFile )
-    {
-        this.systemPropertiesFile = systemPropertiesFile;
-    }
+    public abstract void setSystemPropertiesFile( File systemPropertiesFile );
```

Comment:
Several days ago I committed a unit test whih simply fetched the value to this field using the PowerMock.
Pls utilize the same idea and try to cover the tests on the getters and setters you added. Pls try to cover the `catch` block with the exception.
Now the coverage slowly grows and it goes with all the effort but it is positive thing.

Reply:
Not clear. Could you point the test code which should be repeated in this PR?

Reply:
Not to repeat.
Only my hint to make your life easier :-)
Example in the commit `80659dc4`:

```
    public void testShouldGetEnv()
    {
        SurefirePlugin plugin = new SurefirePlugin();
        plugin.setExcludedEnvironmentVariables( new String[] { "ABC", "KLM" } );
        assertThat( plugin.getExcludedEnvironmentVariables() )
                .hasSize( 2 )
                .contains( "ABC", "KLM" );
    }
```